src/main/java/org/zalando/jackson/datatype/money/MonetaryAmountSerializer.java

```diff
@@ -24,21 +24,34 @@
 import com.fasterxml.jackson.databind.JsonSerializer;
 import com.fasterxml.jackson.databind.SerializerProvider;
 
+import javax.annotation.Nullable;
 import javax.money.CurrencyUnit;
 import javax.money.MonetaryAmount;
+import javax.money.format.MonetaryAmountFormat;
 import java.io.IOException;
 import java.math.BigDecimal;
 
 public final class MonetaryAmountSerializer extends JsonSerializer<MonetaryAmount> {
 
+    private final MonetaryAmountFormat format;
```

Comment:
> Instances of this class are not required to be thread-safe

Good catch!

> instead of having one single locale for the singleton ObjectMapper

I think the Locale that one can configure in Jackson's SerializationConfig is exactly that, a *single locale for the singleton ObjectMapper*.